component.js

```diff
@@ -102,6 +103,7 @@ function factory (options) {
   var _isJsx = !!options.jsx;
   var _hiddenCursorField = options.cursorField || '__singleCursor';
   var _isNode = options.isNode || isNode;
+  var _cached = cached.withDefaults(_shouldComponentUpdate);
```

Comment:
This isn't right? This should probably just be `cached.withDefaults(options)`, given that the function used (currently `isEqualProps`) is descriptive for this use case and it's clear that changing default on Omniscient also changes default on cached. `isEqualProps` won't work for the reasons described in a previous comment.